cli.py

```diff
@@ -0,0 +1,36 @@
+import typer
+
+from test import test_yolo
+from train import train_yolo
+
+
+app = typer.Typer()
+
+
+@app.command()
+def train(
+    pre_trained_model_path: str = typer.Option(
+        "--path_to_pretrained_model", help="Path to pre-trained model"
+    ),
+    data_to_be_trained_config_path: str = typer.Option(
+        "--data_to_be_trained_config_path", help="Path to dataset to be train"
```

Comment:
We can rename it to `dataset_config_path`. Also update the `help` to reflect that